public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/QueryAndExpressionsStep.tsx

```diff
@@ -90,19 +85,21 @@ export function areQueriesTransformableToSimpleCondition(
   if (dataQueries.length !== 1) {
     return false;
   }
+  const reducerRemovedOk =
```

Comment:
nit: how about changing the name to `hasNoReduceExpression` or something like that?
Should we also actually check if the single expression query is not `reduce`?

I got a bit confused because reduce expression and redux reducer have a very similar name 😄

Reply:
I will change it to `singleReduceExpressionInInstantQuery` => this const is true when we have only one reduce expression and we have an instant query.